Swiftfin/Views/PagingLibraryView/PagingLibraryView.swift

```diff
@@ -251,6 +256,28 @@ struct PagingLibraryView<Element: Poster>: View {
         .proxy(collectionVGridProxy)
     }
 
+    @ViewBuilder
+    private func contentLetterBarView(content: some View) -> some View {
+        if letterPickerEnabled {
+            switch letterPickerOrientation {
+            case .trailing:
+                HStack(spacing: 0) {
+                    content
+                        .frame(maxWidth: .infinity)
+                    LetterPickerBar(viewModel: viewModel.filterViewModel!)
```

Comment:
One thing I missed earlier is that not all libraries _have_ filters, which is why the `FilterViewModel` is optional. For example, the libraries reached from `HomeView` don't have filters and this crashes.

Reply:
Shoot, I had something in there for that. I'm not sure why I got rid of that but I had:

if letterPickerEnabled && viewModel.filterViewModel != nil {...

I've re-added this.